src/plugins.js

```diff
@@ -1,8 +1,8 @@
 import Inert from 'inert';
 import HapiPino from 'hapi-pino';
 import CsrfPlugin from 'electrode-csrf-jwt';
+import config from 'mwp-cli/src/config';
```

Comment:
i think we have a weird dependency relationship between consumers, meetup-web-platform, and mwp-cli. It strikes me as an unwanted side effect that mwp-cli should need to be updated to supply config to meetup-web-platform, particularly since the value itself is coming from the consumer. 

Taking a look at meetup-web-platform, the other place where this occurs is providing the browser js output dir to the service worker code. This value could be provided by the consumer repo. Both values could be read directly from the consumer repos and the dependency between meetup-web-platform and mwp-cli could be removed.